src/content/docs/en/tutorial/1-setup/index.mdx

```diff
@@ -8,6 +8,9 @@ description: >-
   Prepare your development environment, and create and deploy your first Astro
   site
 i18nReady: true
+head:
+  - tag: title
+    content: 'Build a blog tutorial: Unit 1 - Setup | Docs'
```

Comment:
Just a quick note that we want this done for **all six** of the tutorial units to match this format!

If you're reading this PR, you could be contributing those changes!